README.md

```diff
@@ -93,6 +93,16 @@ Stairway is designed to provide atomic operations for one instance of one servic
 global or cross-service state. Therefore, it is up to the application or service to implement concurrency control on 
 its objects.
 
+### Context Awareness and Logs
+Stairway leverages the underlying logging system's mapped diagnostic context (MDC) if available.
+
+MDC manages contextual information on a per-thread basis: as Stairway submits a Flight for processing,
+it **passes along the MDC of the calling thread** so that context isn't lost.  It also further populates
```

Comment:
Propagating the MDC from the calling thread makes me want to designate this as a minor version bump at minimum.  Default behavior in this repo is patch bumps.

Reviewers, WDYT?

Reply:
I agree with a more significant version bump. Though with all the dependency upgrades underway, I was already thinking we would do a major version bump very soon. Perhaps just do a major bump with this change?

Reply:
Good idea. I'll make it a major one.

Reply:
Added #major to the PR title, which will get pulled into the commit message.